### DISCUSSION TITLE
PyREP support for Workbench

### DISCUSSION THREAD

**Discussion by mkvani** [ansys/pyhps#223] [Q&A] [answered]
Is PyREP Ansys product agnostic? 
If not, does/will it support Ansys Workbench job submission? What is the timeline for this?

Thank you,
Manoj

**Comment by FedericoNegri** [comment, marked as answer]
Hi @mkvani
PyREP is product agnostic, it simply wraps around the REP (now renamed to Ansys HPC Platform Services) API.

What we provide are examples showing how to submit jobs for different solvers using PyREP scripting. We don't have any for Workbench, but we could prepare one together if you're interested.

Also, Workbench offers a native integration with REP, to submit both design points and project updates. It's currently available under a feature flag.

**Reply by mkvani** [reply]
Thank you @FedericoNegri for your quick response. 
I will connect with you separately to discuss this. 

Best regards,
Manoj
